src/site/shared/containers/ItemNav/index.tsx

```diff
@@ -183,12 +221,13 @@ export const ItemNav = <D,>({ info, config, additionalData,
     const sections = (side === "left") ? config.sections : sectionsBottom;
 
     return (<>
+        {deletingItemPreview()}
         <div className="itemnav__preview"
              style={{
-                display: (curItemImg ? "initial" : "none"),
-                left: pos.x,
-                top: pos.y,
-             }}>
+                 display: (curItemImg ? "initial" : "none"),
+                 left: pos.x,
+                 top: pos.y,
+                }}>
```

Comment:
Not sure if linting said to change this spacing, but if it didn't then it should be what it was before